sycl/test-e2e/Graph/graph_common.hpp

```diff
@@ -432,3 +432,16 @@ class Barrier {
   std::condition_variable cv;
   std::size_t threadNum;
 };
+
+template <typename T>
+bool inline check_value(const size_t index, const T &Ref, const T &Got,
+                        const std::string &VariableName) {
+  if (Got != Ref) {
+    std::cerr << "Unexpected value at index " << index << " for "
```

Comment:
All other end-to-end tests use `cout` for this kind of output. Any needs to align here?

Reply:
Used `cout` based on personal preference for where error messages, but being consistent is more important so updated to `std::cout`.